bundle/manifests/registry-operator.clusterserviceversion.yaml

```diff
@@ -615,7 +615,7 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
-                image: quay.io/devfile/registry-operator:next
+                image: quay.io/devfile/registry-operator:v0.1.3
```

Comment:
is the version `v0.1.3` to match the updated patch version number of should it be `v0.1.2` as mentioned in the PR description?

Reply:
> is the version `v0.1.3` to match the updated patch version number of should it be `v0.1.2` as mentioned in the PR description?

@kim-tsao The new patch version for these PR changes, the link to the community operators PR currently states 0.1.2 but will be changed to 0.1.3 once these changes are applied to it.